src/server/nyplApiClient/index.ts

```diff
@@ -0,0 +1,128 @@
+import NyplApiClient from "@nypl/nypl-data-api-client"
+import aws from "aws-sdk"
+
+interface KMSCache {
+  clients: string[]
+  clientId: string
+  clientSecret: string
+}
+
+const config: any = {
+  api: {
+    platform: {
+      development:
+        process.env.PLATFORM_API_BASE_URL ||
+        "https://qa-platform.nypl.org/api/v0.1",
+      production:
+        process.env.PLATFORM_API_BASE_URL ||
+        "https://platform.nypl.org/api/v0.1",
+    },
+    // The 'discovery' base URL should use DISCOVERY_API_BASE_URL if set,
```

Comment:
What is the difference between DISCOVERY_API_BASE_URL and PLATFORM_API_BASE_URL?

Reply:
I'm unsure if we should have a fallback here. In the case that we are missing such a crucial env var, I think it would be better to have the app break than to mistakenly have the production serving QA data.

Reply:
That's a good point, again unsure if there are historical reasons for that but happy to consolidate the team agrees it's not necessary anymore.

Reply:
Context is in this PR: https://github.com/NYPL/discovery-front-end/pull/2109

Reply:
ah ok i didnt realize that this was mostly just what was already happening in the DFE. i must not have the most recent version on my local. makes sense!